presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -447,6 +447,83 @@ Property Name                                        Description
                                                      defaults to ``20``.
 ==================================================== ============================================================
 
+Performance Tuning Configuration Properties
+-------------------------------------------
+
+The following table describes performance tuning properties for the Hive
+connector.
+
+.. warning::
+
+   Performance tuning configuration properties are considered expert-level
+   features. Altering these properties from their default values is likely to
+   cause instability and performance degradation. We strongly suggest that
+   you use them only to address non-trivial performance issues, and that you
+   keep a backup of the original values, if you change them. 
+
+.. list-table:: Hive performance tuning configuration properties
+    :widths: 30, 50, 20
+    :header-rows: 1
+
+    * - Property name
+      - Description
+      - Default value
+    * - ``hive.domain-compaction-threshold``
+      - Sets the number of transactions to act as threshold. Once reached the
+        connector initiates compaction of the underlying files and the delta
+        files. A higher compaction threshold means reading less data from the
+        underlying data source, but a higher memory and network consumption.
+      - 100
+    * - ``hive.max-outstanding-splits``
+      - The target number of buffered splits for each table scan in a query,
+        before the scheduler tries to pause.
+      - 1000
+    * - ``hive.max-splits-per-second``
+      - Sets the maximum number of splits used per second to access underlying
```

Comment:
Change description to

> The maximum number of splits generated per second per table scan. This can be used to reduce the load on the storage system. By default, there is no limit, which results in Presto maximizing the parallelization of data access.